serve/graph/schema/filter/transaction_filter.graphql

```diff
@@ -0,0 +1,240 @@
+  """
+  from_index: Int
+
+  """
+  Maximum Transaction index up to which to fetch, exclusive. Ensures a limit on the ordering range for Transaction queries.
+  """
+  to_index: Int
+
+  """
+  Minimum `gas_wanted` value to filter Transactions by, inclusive. Filters Transactions based on the minimum computational effort declared.
+  """
+  from_gas_wanted: Int
+
+  """
+  Maximum `gas_wanted` value for filtering Transactions, exclusive. Limits Transactions based on the declared computational effort.
+  """
+  to_gas_wanted: Int
+
+  """
+  Minimum `gas_used` value to filter Transactions by, inclusive. Selects Transactions based on the minimum computational effort actually used.
+  """
+  from_gas_used: Int
+
+  """
+  Maximum `gas_used` value for filtering Transactions, exclusive. Refines selection based on the computational effort actually consumed.
+  """
+  to_gas_used: Int
+
+  """
+  Hash from Transaction content in base64 encoding. If this filter is used, any other filter will be ignored.
+  """
+  hash: String
+
+  """
+  Transaction's message to filter Transactions.
+  """
+  message: TransactionMessageInput
+
+  """
+  `memo` value to filter Transaction's memo.
+  """
+  memo: String
+}
+
+"""
+Input for filters by transaction message.
+"""
+input TransactionMessageInput {
+  """
+  The type of transaction message.
+  """
+  type_url: MessageType
+
+  """
+  The route of transaction message.
+  """
+  route: MessageRoute
+
+  """
+  `TransactionBankMessageInput` represents input parameters required when the message router is `bank`.
+  """
+  bank_param: TransactionBankMessageInput
+
+  """
+  `TransactionVmMessageInput` represents input parameters required when the message router is `vm`.
+  """
+  vm_param: TransactionVmMessageInput
+}
+
+"""
+`TransactionBankMessageInput` represents input parameters required when the message router is `bank`.
+"""
+input TransactionBankMessageInput {
+  """
+  send represents input parameters required when the message type is `send`.
+  """
+  send: BankMsgSendInput
+}
+
+"""
+`BankMsgSendInput` represents input parameters required when the message type is `send`.
+"""
+input BankMsgSendInput {
+  """
+  the bech32 address of the fund sender.
+  """
+  from_address: String
+
+  """
+  the bech32 address of the fund receiver.
+  """
+  to_address: String
+
+  """
+  the denomination and amount of fund sent ("<amount><denomination>").
+  """
+  amount: String
+}
+
+"""
+`TransactionVmMessageInput` represents input parameters required when the message router is `vm`.
+"""
+input TransactionVmMessageInput {
+  """
+  `MsgCallInput` represents input parameters required when the message type is `exec`.
+  """
+  m_call: MsgCallInput
+
+  """
+  `MsgAddPackageInput` represents input parameters required when the message type is `add_package`.
+  """
+  m_addpkg: MsgAddPackageInput
+
+  """
+  `MsgRunInput` represents input parameters required when the message type is `run`.
+  """
+  m_run: MsgRunInput
```

Comment:
Those names are the endpoint contents of the typeUrl when creating the transaction.
ex) vm.m_call, vm.m_addpkg ...

But now it seems better to specify them as `exec`, `add_package`, `run`. Because we already use these values.

Reply:
Yeah, LGTM